### ISSUE TITLE
Polymer 2.0

### ISSUE THREAD

**Issue by kelvinwatson** [vguillou/parallax-element#8]
This component doesn't seem to be working in Polymer 2.0. I copied the demo code verbatim and nothing appears on the screen. There is no console output indicating an error.

**Comment by vguillou** [comment]
Yes, it is a Polymer 1 component. I'm not planning to upgrade it, if someone wants to submit I pull request, I'd be happy to take a look!
